<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmWk3260</class>
 <widget class="QDialog" name="frmWk3260">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>266</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>参数设置</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/wk.ico</normaloff>:/wk.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	
	background-color: rgb(148, 148, 248);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="title">
        <string>测试参数</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>项目1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmbFuction1">
          <property name="minimumSize">
           <size>
            <width>67</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>L</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>C</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Z</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>项目2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cmbFunction2">
          <property name="minimumSize">
           <size>
            <width>67</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Q</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>D</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>R</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Bias 控制</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Bias状态</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmbBiasChioce">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Norm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Boost</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Iswp</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cmbAddBiasSpeed">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>OFF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OnMax</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Bias 值</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btnBiasValue">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1.00A</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Bias 开关</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="btnBiasOnOFF">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>OFF</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="title">
        <string>电平&amp;&amp;频率</string>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>电平</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnLevel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1.00V</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>频率</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btnFrequency">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>100kHz</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="title">
        <string>速度&amp;&amp;等效电路</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>测试速度</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmbSpeed">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>最快</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>快速</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>中速</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>慢速</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>等效电路</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cmbEqucct">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>串联</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>并联</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnUpdate">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
